data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/BeaconRestApi.java

```diff
@@ -122,6 +123,7 @@ private void addBeaconHandlers(final DataProvider dataProvider) {
 
   private void addValidatorHandlers(DataProvider dataProvider) {
     ChainDataProvider provider = dataProvider.getChainDataProvider();
+    app.post(PostValidatorDuties.ROUTE, new PostValidatorDuties(provider, jsonProvider));
```

Comment:
int: I'm thinking it might be better to group the POSTs down the bottom initially, potentially at some point we might break them into separate setup functions...

Reply:
do you want a new method or just put it at the bottom of the list?

Reply:
done